message_ix_models/tools/costs/projections.py

```diff
@@ -393,15 +393,23 @@ def create_message_outputs(
         .rename(columns={"inv_cost": "inv_cost_2100", "fix_cost": "fix_cost_2100"})
     )
 
+    dims = [
+        "scenario_version",
+        "scenario",
+        "message_technology",
+        "first_technology_year",
+        "region",
+    ]
+
     df_merge = (
         (
             df_prod.merge(
                 val_2020,
-                on=["scenario_version", "scenario", "message_technology", "region"],
+                on=dims,
```

Comment:
I think if we remove the trailing comma, `black` or `ruff` will automatically reflow this method call into a single line. I will try to push a commit that does this.